excludeFindBugsFilter.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<FindBugsFilter>
```

Comment:
Naming is a little bit confusing I think. Given that this is a config file of `spotBugs`, shouldn't we call it `spotBugsFilter.xml` instead?
Though the XML tag name has remained "FindBugs", so I'm not sure.

Reply:
@anmolnar  this name is very common, as many project migrated from findbugs.
I can change, no problem. But I don't feel strong about it.

Reply:
Leave it, I'm fine with that.